packages/dataviews/src/filter-and-sort-data-view.ts

```diff
@@ -61,7 +61,7 @@ export function filterSortAndPaginate< Item >(
 		} );
 	}
 
-	if ( view.filters?.length > 0 ) {
+	if ( view.filters && view.filters?.length > 0 ) {
```

Comment:
What does this change?

Reply:
I made view.filters optional that way folks don't have to define it if they don't have filters.

Reply:
Yeah, I meant why the previous `view.filters?.length` wasn't enough..